### ISSUE TITLE
ConnexionPrometheusMetrics - less metrics?

### ISSUE THREAD

**Issue by larrycai** [rycus86/prometheus_flask_exporter#67]
I tried the `connexion-pydantic` example in docker env.

```
cd connexion-pydantic
docker build -f Dockerfile -t connexion-pydantic ../../.
docker run -d --name connexion-pydantic -p 4000:4000 connexion-pydantic
````

And when I checked the `http://localhost:4000/metrics`, i only got some metrics

````
# HELP python_gc_objects_collected_total Objects collected during gc
# TYPE python_gc_objects_collected_total counter
python_gc_objects_collected_total{generation="0"} 1112.0
...
test_by_status_sum{code="200"} 0.00013733200103160925
# HELP test_by_status_created Test Request latencies by status
# TYPE test_by_status_created gauge
test_by_status_created{code="200"} 1.5953459881785243e+09
````

If it is normal flask application, I got lots of metrics with `flask_` like

````
# HELP flask_http_request_duration_seconds Flask HTTP request duration in seconds
# TYPE flask_http_request_duration_seconds histogram
flask_http_request_duration_seconds_bucket{le="0.005",method="GET",path="/",status="200"} 1.0
flask_http_request_duration_seconds_bucket{le="0.01",method="GET",path="/",status="200"} 1.0
flask_http_request_duration_seconds_bucket{le="0.025",method="GET",path="/",status="200"} 2.0
flask_http_request_duration_seconds_bucket{le="0.05",method="GET",path="/",status="200"} 2.0
flask_http_request_duration_seconds_bucket{le="1.0",method="GET",path="/",status="200"} 2.0
````

Anything I missed here

**Comment by rycus86** [comment]
Hi @larrycai 

That example was provided to me if I remember correctly and has `export_defaults=None` set in the `main.py` file.
If you drop that argument, or change it to `export_defaults=True`, it should work as you expected it above.

**Comment by rycus86** [comment]
This actually fails currently, as I didn't realize we pass around a Connexion `FlaskApp` object, not a Flask application.

**Comment by rycus86** [comment]
This should be fixed in `0.15.4`, Connexion should now work with `export_defaults=True`

**Comment by larrycai** [comment]
Yes, i tested as well with latest version and update line with `export_defaults=True`

````
metrics = ConnexionPrometheusMetrics(app, export_defaults=True)
````
